test/test_main.py

```diff
@@ -87,7 +87,43 @@ def test_file_not_given(self, mock_stdout):
     def test_wrong_platform(self, mock_stdout):
         fcli = FlasherCLI(["flash", "-i", "test/helloworld.bin", "-t", "K65G"])
         self.assertEqual(fcli.execute(), 10)
-        self.assertEqual(mock_stdout.getvalue(), "Not supported platform: K65G\nSupported platforms: [u'NRF51822', u'K64F', u'NRF51_MICROBIT', u'NRF51_DK', u'NUCLEO_F401RE']\n")
+        self.assertIn(mock_stdout.getvalue(), "Not supported platform: K65G\nSupported platforms: "
```

Comment:
this is not good approach.. its enough to verify first sentence,, otherwise we have to update this every time when we add more target support...